CMakeLists.txt

```diff
@@ -1,10 +1,20 @@
 cmake_minimum_required(VERSION 3.12)
 project(
   Dispenso
-  VERSION 0.1.0
+  VERSION 1.0.0
   DESCRIPTION "Dispenso is a library for working with sets of parallel tasks"
   LANGUAGES CXX)
 
+if ("${CMAKE_SOURCE_DIR}" STREQUAL "${CMAKE_CURRENT_SOURCE_DIR}")
+  set(DISPENSO_STANDALONE TRUE)
+else()
+  set(DISPENSO_STANDALONE FALSE)
+endif()
+
+if (DISPENSO_STANDALONE)
+  include(GNUInstallDirs)
```

Comment:
Does this have any implications on Windows?  Sorry, I'm not great with CMake (as you may have guessed).

Reply:
Also, is there a downside with simply always using this layout instead of using standalone?  The project is still new as far as cmake use is concerned, so changing the default layout is probably okay.

Reply:
> Does this have any implications on Windows?

The short answer is no -- `GNUInstallDirs` just sets up some standard CMake variables in the cache for controlling where headers, libs, apps, docs, etc. all go. The "GNU" part is only the origin of the defaults, which is a perfectly usable layout for installing on Windows (this is what I do in all the libraries I've ever worked on).

> Also, is there a downside with simply always using this layout instead of using standalone? 

The idea with `DISPENSO_STANDALONE` is simply that if someone uses dispenso directly with `add_subdirectory()` (probably as a git submodule), in which case the parent project should control the install. In other words, `DISPENSO_STANDALONE` is only guarding against when someone's parent project does `make install` and unexpectedly dispsenso installs itself too (in a potentially conflicting destination). Thus I would expect that a parent project may or may not want `GNUInstallDirs` included...but it's relatively harmless to always include it anyway.

Reply:
Thanks for the explanation.  This looks good